<launch>
    <node pkg="rviz"    type="rviz" name="rviz" args="-d $(find argo_tracking)/rviz_config/tracking_result_viz.rviz"/>

    <arg name="scene_name"              default="c6911883-1843-3727-8eaa-41dc8cda8993"/>
    <arg name="tracking_result_folder"  default="/home/yellow/argoverse_cbgs_kf_tracker/temp_files/val-split-track-preds-maxage15-minhits5-conf0.3"/>
    <arg name="argo_data_folder"        default="/home/yellow/argodataset/argoverse-tracking/sample"/>
    <arg name="detection_folder"        default="/home/yellow/argoverse_cbgs_kf_tracker/argoverse_detections_2020/training"/>


    <node pkg ="argo_tracking"   type="viz_track_result" name="viz_result"   output="screen">
        <param name ="log_path"             type="string"       value="$(arg argo_data_folder)/$(arg scene_name)"/>
        <param name ="label_path"           type="string"       value="$(arg tracking_result_folder)/$(arg scene_name)"/>
        <param name ="det_path"             type="string"       value="$(arg detection_folder)/$(arg scene_name)"/>
        <param name ="play_rate"            type="int"          value="5"/>
        <param name ="id_type"              type="string"       value="uuid"/>
        <param name ="rviz_wating_time"     type="double"       value="3.0"/>
    </node>

</launch>
